RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -175,11 +175,6 @@ boolean tryAddItemToMaximize(slot s, item it)
 
 string defaultMaximizeStatement()
 {
-	if(in_pokefam())
-	{
-		return pokefam_defaultMaximizeStatement();
-	}
```

Comment:
what is the logic of removing this?

Reply:
Seemed to me that default maximizer worked better for areas like +item, increased NC, and +ML for leveling.  The default pokefam one was minimal

Reply:
Those are still used. the process is:
1. resetMaximize() is called
2. it sets `res = defaultMaximizeStatement();`
3. resetMaximize() adds a bunch of stuff as needed

the lines you deleted only changed the output of defaultMaximizeStatement() in step 2.
without this bit you get a default statement of
`5item,meat,0.5initiative,0.1da 1000max,dr,0.5all res,1.5mainstat,mox,-fumble,mox,0.4hp,0.2mp 1000max,1.5weapon damage,0.75weapon damage percent,1.5elemental damage,10exp,5" + my_primestat() + " experience percent`

the only bits of it relevant to pokefam are `5item,meat,10exp,5" + my_primestat() + " experience percent`
which are exactly the 4 things returned by `pokefam_defaultMaximizeStatement();`

the default statement is meant to be stuff general to all location. for specific location stuffs we then do things like +ML, special item restriction/addition, and location specific desires. those are all handled in resetMaximize(). just scroll down a bit to see it.

as such this function should be un-deleted

Reply:
Thank you, taltamir.  I added all of this back in